lib/index.js

```diff
@@ -103,10 +103,14 @@ util.inherits(Encoder, Transform)
 Encoder.prototype._transform = function(message, enc, cont) {
   var length = new Buffer(this.opts.lengthSize)
   this.setLength(length, message.length)
-  this.push(Buffer.concat([length, message]))
+  this._pushFrameData([length, message])
   cont()
 }
 
+Encoder.prototype._pushFrameData = function(bufs) {
```

Comment:
No specific reason - happy to change it. Was thinking it might mean we could change the implementation (e.g. if we required 3 buffers) without the override having to be changed.